src/wallet/wallet.ts

```diff
@@ -1010,7 +1010,7 @@ class HathorWalletServiceWallet extends EventEmitter implements IHathorWallet {
     this.failIfWalletNotReady();
     const newOptions = {
       token: '00',
-      changeAddress: null,
+      changeAddress: undefined,
```

Comment:
Passing a `changeAddress` as null was not expected by the `sendManyOutputsTransaction` method and had to be changed to `undefined`, otherwise the build phase would break.

The tests had to be changed accordingly. Since this is a static type issue, it's not expected that it will cause any issues with the wallet-lib consumers.